temporal/api/cloud/namespace/v1/message.proto

```diff
@@ -124,3 +125,25 @@ message Namespace {
     // Will not be set if the namespace has never been modified.
     google.protobuf.Timestamp last_modified_time = 11;
 }
+
+enum FailoverType {
+    FAILOVER_TYPE_UNSPECIFIED = 0;
+    FAILOVER_TYPE_GRACEFUL = 1;
+    FAILOVER_TYPE_FORCEFUL = 2;
+}
+
+enum FailoverStatus {
+    FAILOVER_STATUS_UNSPECIFIED = 0;
+    FAILOVER_STATUS_SUCCEEDED = 1;
+    FAILOVER_STATUS_FAILED = 2;
+}
+
+message FailoverRecord {
+    FailoverType failover_type = 1;
+    temporal.api.cloud.region.v1.Region source_region = 2;
```

Comment:
Do we need a source_region? Isn't it implied by the active region of the namespace?
https://github.com/temporalio/api-cloud/blob/main/temporal/api/cloud/namespace/v1/message.proto#L116